docs/resources/job.md

```diff
@@ -405,7 +405,7 @@ resource "databricks_job" "sql_aggregation_job" {
 }
 ```
 
-### Exported attributes
```

Comment:
Have you replaced all the instances of this in the doc?

Reply:
yes, it was only one place with such a name - everything else uses "Attribute Reference"